<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="pages">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>821</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_inserir">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>0</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Funcionários</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Projetos</string>
      </property>
     </widget>
     <widget class="QSplitter" name="splitter_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>381</width>
        <height>41</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Nome:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="input_p_nome"/>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>390</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>(Caso esteja em andamento, mantenha 01/01/2000)</string>
      </property>
     </widget>
     <widget class="QSplitter" name="splitter_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Tempo estimado(meses):</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="input_p_meses"/>
     </widget>
     <widget class="QSplitter" name="splitter_8">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>220</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Valor estimado:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="input_p_valor_est"/>
     </widget>
     <widget class="QSplitter" name="splitter_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Funcionário responsável:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="input_p_n_func"/>
     </widget>
     <widget class="QTextBrowser" name="log_inserir">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>40</y>
        <width>371</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Resultado das operações:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_inserir_f">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>110</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Inserir</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_p_inserir">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>300</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Inserir</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_limpar_p_inserir">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>300</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Limpar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_limpar_f_inserir">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>60</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Limpar</string>
      </property>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Nome</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="input_f_nome"/>
     </widget>
     <widget class="QSplitter" name="splitter_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>90</y>
        <width>133</width>
        <height>41</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Salário:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="input_f_salario"/>
     </widget>
     <widget class="QSplitter" name="splitter_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Número Funcional (Opcional):</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="input_f_n_func"/>
     </widget>
     <widget class="QSplitter" name="splitter_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>370</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Data Término:</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="data_p_termino">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Data Início:</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="date_p_inicio">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_editar">
     <widget class="QTextBrowser" name="log_alt">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>20</y>
        <width>311</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Editar / Remover</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_select_alt">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>420</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>(Caso esteja em andamento, mantenha 01/01/2000)</string>
      </property>
     </widget>
     <widget class="QSplitter" name="splitter_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Funcionário responsável:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="input_p_n_func_alt"/>
     </widget>
     <widget class="QRadioButton" name="rbtn_f_alt">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Funcionário</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtn_p_alt">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Projeto</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_rm_alt">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>80</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Remover</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_f_nome_alt">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>301</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_f_salario_alt">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>210</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="output_f_n_func_alt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Projeto</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>190</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Salário:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Número Funcional (Não editável):</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Funcionário</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nome:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="output_p_nome_alt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>341</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_f_alterar">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>160</y>
        <width>101</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Alterar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_p_alterar">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>360</y>
        <width>101</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Alterar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>60</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Deseja remover?</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Data Início:</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="date_p_inicio_alt">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>370</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>68</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Data Término:</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="data_p_termino_alt">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>400</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nome (Não editável):</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>310</y>
        <width>120</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Tempo estimado(meses):</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_p_meses_alt">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>329</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_p_valor_est_alt">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>329</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>310</y>
        <width>74</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Valor estimado:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_buscas">
     <widget class="QTreeWidget" name="table_busca">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>801</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>0</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Buscas</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Listar Funcionários
(Salário decrescente)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>80</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar Endereço
 de memória</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>30</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Número Funcionário:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>50</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>50</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Listar Funcionários
com projetos
em andamento
atribuídos</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>50</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Listar projetos
de mais de R$500k
em andamento</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>50</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Listar projetos
atrasados ou
terminados com atraso</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_tela_inserir">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>INSERIR DADOS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_tela_alterar">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>EDITAR/REMOVER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_tela_buscar">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>BUSCAS</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
